src/pcapi/settings.py

```diff
@@ -74,3 +74,24 @@
 #   - mailjet
 #   - log
 SEND_RAW_EMAIL_BACKEND = os.environ.get("SEND_RAW_EMAIL_BACKEND", "mailjet").lower()
+
+
+# ALGOLIA
+ALGOLIA_API_KEY = os.environ.get("ALGOLIA_API_KEY")
+ALGOLIA_APPLICATION_ID = os.environ.get("ALGOLIA_APPLICATION_ID")
+ALGOLIA_INDEX_NAME = os.environ.get("ALGOLIA_INDEX_NAME")
+ALGOLIA_TRIGGER_INDEXATION = os.environ.get("ALGOLIA_TRIGGER_INDEXATION", "0")
```

Comment:
```suggestion
ALGOLIA_TRIGGER_INDEXATION = bool(int(os.environ.get("ALGOLIA_TRIGGER_INDEXATION", "0")))
```

... et l'utiliser comme un booléen [ici](https://github.com/pass-culture/pass-culture-api/pull/1672/files#diff-192eeba949d8d8eace539c9a09d23007bd68b752570eadc07389dfeb2078b367R12) ?

(C'est un peu séparé de ce commit, je te l'accorde.)